app/views/static_pages/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= javascript_pack_tag 'application' %>
+<%= react_component("App") %>
```

Comment:
- [ ] Good job so far 👍 but the static view does not display and instead gives the following error. Kindly address this issue 💯 
- Error:
![Screenshot 2022-07-09 191426](https://user-images.githubusercontent.com/85113567/178118162-d2422b38-0c54-46aa-bbed-130302aa25a2.png)